odoo/addons/base/models/ir_ui_menu.py

```diff
@@ -55,7 +55,7 @@ def _get_full_name(self, level=6):
         if self.parent_id:
             return self.parent_id._get_full_name(level - 1) + MENU_ITEM_SEPARATOR + (self.name or "")
```

Comment:
```suggestion
            return (self.parent_id._get_full_name(level - 1) or '') + MENU_ITEM_SEPARATOR + (self.name or "")
```
We should apply fix here since we got self.parent_id._get_full_name(level - 1)

Reply:
Thanks, i have made changes